### DISCUSSION TITLE
Is dialogic 2 usable? Is it like HALF stable?

### DISCUSSION THREAD

**Discussion by UNiV135** [dialogic-godot/dialogic#1884] [Q&A]
im just curious because i want to make my game with this and a bit worried

**Comment by Jowan-Spooner** [comment]
Hey, @UNiV135 thanks for you interest in dialogic 2.

### Short answer: 
Yes.

### Long answer: 
Yes, but...
Obviously dialogic is still in alpha, which is probably why you asked this question. Dialogic already works pretty well with godot 4.1 and beyond. It already has lot's of features and allows quickly creating dialogs or event small VN's. While it has some bugs, it's not "buggy as hell", instead most things work pretty well already.
*However*, we want dialogic 2 to be really good before we commit to "no more breaking changes". While the breaking changes from alpha to alpha are generally pretty small, there are some systems that are not finished yet. If you are willing to make smaller adjustments here and there whenever you update then you can probably already start using it. Of course we reserve the right to breaking changes if we think it will make the final version better.
Another thing to consider is that there is currently only little documentation (again because we want to finish dialogics systems first, before polishing around it).

Hope this helps.
